tests/edge-cases/ash-02.txt

```diff
@@ -0,0 +1,2 @@
+brun '(ash (q 1))'
+FAIL: ash requires 2 args (q)
```

Comment:
One option is the s-expression in the error message can be decompiled without keywords (since it generally just includes already-evaluated expressions).